CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.14..3.18)
 project(kphp
         VERSION 1.0.1
+        LANGUAGES CXX
```

Comment:
Previously here was a room for misconfiguration between c and cpp compilers like in our clang pipeline:
![image](https://user-images.githubusercontent.com/14000874/134676686-499ff9c9-45c2-4efd-a34d-99b3f3db1b27.png)
Since we don't use c compiler I just remove  it.